docs/docsite/rst/plugins/lookup.rst

```diff
@@ -31,30 +23,29 @@ Lookups are an Ansible-specific extension to the Jinja2 templating language.
 Enabling lookup plugins
 -----------------------
 
-You can activate a custom lookup by either dropping it into a ``lookup_plugins`` directory adjacent to your play, inside a role, or by putting it in one of the lookup directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>`.
+Ansible enables all lookup plugins it can find. You can activate a custom lookup by either dropping it into a ``lookup_plugins`` directory adjacent to your play, inside a role, or by putting it in one of the lookup directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>`.
```

Comment:
Ooh, good catch. Lookup plugins don't need to be whitelisted or otherwise actively enabled but those locations should take account of collections. I'll update it to read:

``
You can activate a custom lookup by either dropping it into a ``lookup_plugins`` directory adjacent to your play, inside the ``plugins/lookup/`` directory of a collection you have installed, inside a standalone role, or in one of the lookup directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>`.
```